client/dashboard/profile-wizard/steps/industry.js

```diff
@@ -31,9 +31,11 @@ class Industry extends Component {
 		this.state = {
 			error: null,
 			selected: profileItems.industry || [],
+			textInputListContent: [],
```

Comment:
This is an array, but it looks to be used like an object later?

Reply:
Totally! I should have declared an object instead of an array.